src/lib.rs

```diff
@@ -442,9 +450,12 @@ impl DoubleArrayTrie {
 #[cfg(test)]
 mod tests {
     use super::*;
+    #[cfg(feature = "serialization")]
     use std::fs::File;
+    #[cfg(feature = "serialization")]
```

Comment:
Is

    #[cfg(feature = "serialization")]
    use std::{
        fs::File;
        io::BufReader;
    }

better?

Reply:
I think so, addressed.